src/routers/projects.js

```diff
@@ -2,8 +2,91 @@ const express = require("express");
 
 const router = express.Router();
 
-router.get("/", function (req, res, next) {
+router.get("/", function (req, res) {
   res.status(200).send({ sucess: true });
 });
 
+router.get("/:projectId/versions", async (req, res) => {
+  const { projectId } = req.params;
+  const accessToken = req.headers.authorization;
+
+  if (accessToken) {
+    try {
+      const getVersions = await fetch(
+        `https://api.figma.com/v1/files/${projectId}/versions`,
+        {
+          method: "GET",
+          headers: {
+            "X-FIGMA-TOKEN": accessToken,
+          },
+        },
+      );
+
+      const responseJson = await getVersions.json();
+      const { status, versions } = responseJson;
+
+      if (status === 400) {
+        return res.status(400).send({
+          result: "error",
+          error: {
+            message: "Bad Request",
+          },
+        });
+      }
+
+      if (status === 403) {
+        return res.status(403).send({
+          result: "error",
+          error: {
+            message: "인증되지 않은 사용자 접근입니다.",
+          },
+        });
+      }
+
+      if (status === 404) {
+        return res.status(404).send({
+          result: "error",
+          error: {
+            message: "존재하지 않는 피그마 파일입니다. 다시 입력해주세요",
```

Comment:
P1: message 텍스트는 동일하게 "입력해주세요" 뒤에 . 온점을 동일하게 추가해주시면 좋을 것 같습니다 👍 

```suggestion
            message: "존재하지 않는 피그마 파일입니다. 다시 입력해주세요.",
```